importer/DeHeDe.py

```diff
@@ -45,24 +51,33 @@ def update_labels(self):
         self.add_label("de", german)
 
     def set_heritage_id(self):
-        self.add_statement("denkXweb", str(self.nummer))
+        described_qual = {"described_at_url": self.registrant_url}
```

Comment:
Not needed in this case I would say since the Property (on Wikidata) autogenerates the url. (See follow-up comment below)

Reply:
Removed it.